### ISSUE TITLE
State of the application

### ISSUE THREAD

**Issue by dot-mike** [nuttyartist/notes#672]
Hi, I'm new around here.. and I discovered your application through researching for a notes taking app. I liked the simplicity of your application and it looked promising. But while glancing over the repo I saw there has been no real feature / fix commits in over 5 months. So naturally I was a bit worried this project was stalled which lead me to look into the open-issues to understand current state of this application.

I came to discover one of the open issues is the locked-down database format #336 which is a drawback for me. In the same issue I also saw an open issue for importing existing data #316 where I discovered a comment from the devleoper nuttyartist stating that this feature is implemented in [Plume](https://github.com/nuttyartist/plume-public/tree/main) which appears to be another note taking app from the developer of this application.

So I would to ask you @nuttyartist , it this application still supported by you? I can see recent development has happened in Plume. And will this application continue to receive feature implementations or only bug / security fixes?

I would appreciate if you could answer these questions as I think they are important for the community and the roadmap. :pray:

**Comment by nuttyartist** [comment]
Hi, thanks for coming by!

First, it's not really a "locked-down database format", it's a very simple SQLite format, that everyone can look into. It requires some technical knowledge, granted, but still I won't call it locked-down, especially since the entire database code is [completely open source](https://github.com/nuttyartist/notes/blob/master/src/dbmanager.cpp).

This project is very much alive. The thing is, I'm very busy with Plume, like you mentioned. But I just released all the changes I made in Plume from Notes in that [repo](https://github.com/nuttyartist/plume-public). If someone would like to bring those improvements and send a PR, I'd be very happy to merge. Nevertheless, I plan to do it myself once I get the time for that. And indeed, even regular users soon will be able to export their data to .txt/.md like you just mentioned.

May I ask why you don't consider using Plume, is it because it is closed source, and you're looking for an open source note-taking app? Even without paying for the Pro version, you get all the features in Notes and far more.

Regarding new features, it will be difficult to add new features at this time unless I add them to Plume, and they share the same code as with Notes. Again, if other people want to contribute, I welcome that.

Thanks again!
